src/pages/signup.tsx

```diff
@@ -0,0 +1,222 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import { useNavigate } from 'react-router-dom';
+
+import TopBar from '../components/TopBar';
+import DropDownBox from '../components/DropDownBox';
+import { usePostJoin } from '../apis/post/usePostJoin';
+
+const team_options = ['GOTCHA', 'SNIFF', 'READY', 'LOCALMOOD', 'SHAREMIND'];
+const part_options = ['FRONTEND', 'BACKEND'];
+
+const Signup = () => {
+  const [emailValue, setEmailValue] = useState('');
+  const [nameValue, setNameValue] = useState('');
+  const [idValue, setIdValue] = useState('');
+  const [teamValue, setTeamValue] = useState('');
+  const [partValue, setPartValue] = useState('');
+  const [passwordValue, setPasswordValue] = useState('');
+  const [passwordCheckValue, setPasswordCheckValue] = useState('');
+  const [isFilled, setIsFilled] = useState(false);
```

Comment:
react hook에도 타입 지정을 할 수 있어서 ts 쓰신 김에 이부분들 각각의 타입에 맞게 지정해보시는 것도 좋을 것 같아요~~~

```suggestion
  const [emailValue, setEmailValue] = useState<string>('');
  const [nameValue, setNameValue] = useState<string>('');
  const [idValue, setIdValue] = useState<string>('');
  const [teamValue, setTeamValue] = useState<string>('');
  const [partValue, setPartValue] = useState<string>('');
  const [passwordValue, setPasswordValue] = useState<string>('');
  const [passwordCheckValue, setPasswordCheckValue] = useState<string>('');
  const [isFilled, setIsFilled] = useState<boolean>(false);
```